pkgs/applications/virtualization/singularity/generic.nix

```diff
@@ -95,14 +96,28 @@ in
   #   "path/to/source/file1" = [ "<originalDefaultPath11>" "<originalDefaultPath12>" ... ];
   # }
   sourceFilesWithDefaultPaths ? { },
-  # Workaround #86349
-  # should be removed when the issue is resolved
-  vendorHash ? _defaultGoVendorArgs.vendorHash,
-  deleteVendor ? _defaultGoVendorArgs.deleteVendor,
-  proxyVendor ? _defaultGoVendorArgs.proxyVendor,
+  # Placeholders for the obsolete workaround of #86349
+  # TODO(@ShamrockLee): Remove after the Nixpkgs 25.05 branch-off.
```

Comment:
(I dropped the ball again...) Well, the branch-offs have happened

Reply:
> (I dropped the ball again...) Well, the branch-offs have happened

but we're still in 2024 yet.

Reply:
Oh, wrong year 👌🏻